src/components/OpenSeason.js

```diff
@@ -0,0 +1,136 @@
+import { Flex, HStack, Select, Image, Spinner, useToast, Button } from "@chakra-ui/react";
+import { useState } from "react";
+import { db } from '../utils/firebase';
+import { collection, query, getDocs, where, updateDoc } from "firebase/firestore";
+import opensznimg from '../assets/openseason-white.svg';
+
+const OpenSeason = ({ arrayOfAlivePlayers, roomID, handleOpenSzn }) => {
+    const [selectedOpenSeasonPlayer, setSelectedOpenSeasonPlayer] = useState('');
+    const [loading, setLoading] = useState(false);
+    const toast = useToast();
+
+    const handleChange = (event) => {
+        setSelectedOpenSeasonPlayer(event.target.value);
+    };
+
+    const handleClick = async () => {
+        handleOpenSzn(selectedOpenSeasonPlayer);
```

Comment:
Hmm so if you don't select an open season player, this will first update the open season targets to be nobody, and then throw an error? Is that order on purpose?